capture/src/main/res/values/colors.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <color name="shutter_photo_tint">@color/alfresco_blue_700</color>
+    <color name="disable_color">@color/alfresco_gray_900_15</color>
```

Comment:
Again you don't need this, see my other comments why, but fyi when you're naming resources generically they probably don't belong in the module you put them in. If you wanted to create a `disable_color` you should properly put in in the `theme` module. If you wanted a similiar use resource here it should probably be named `capture_disabled_color`. `shutter_photo_tint` doesn't have the `capture_` prefix because it's unlikely there is another shutter object in another module outside capture.

Reply:
Done